Library/Homebrew/download_strategy.rb

```diff
@@ -184,7 +184,11 @@ def stage
       with_system_path { buffered_write("bunzip2") }
     when :gzip, :bzip2, :compress, :tar
       # Assume these are also tarred
-      with_system_path { safe_system "tar", "xf", cached_location }
+      if ARGV.verbose?
+        with_system_path { safe_system "tar", "xvf", cached_location }
+      else
+        with_system_path { safe_system "tar", "xf", cached_location }
+      end
```

Comment:
Ideally I'd build an flags list with `flags` and have just the flags list changed by the `if`/`else` and then do `with_system_path { safe_system "tar", flags, cached_location }`